sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -35,102 +19,73 @@
 /** Endpoints for the Orchestration service */
 @RestController
 @Slf4j
+@SuppressWarnings("unused")
 @RequestMapping("/orchestration")
 class OrchestrationController {
   private final OrchestrationClient client = new OrchestrationClient();
+  private final OrchestrationService service = new OrchestrationService();
```

Comment:
(Major)

Spring components can be autowired.

```suggestion
@Autowired
private OrchestrationService service;
```